daemon/stores/base.py

```diff
@@ -24,7 +26,17 @@ def _init_stats(self):
         self._num_del = 0
 
     def add(self, *args, **kwargs) -> 'uuid.UUID':
-        """Add a new element to the store. This method needs to be overridden by the subclass"""
+        """Add a new element to the store. This method needs to be overridden by the subclass
+
+
+        .. #noqa: DAR101"""
+        raise NotImplementedError
+
+    def update(self, *args, **kwargs) -> 'uuid.UUID':
+        """Add a new element to the store. This method needs to be overridden by the subclass
```

Comment:
fix docstring

Reply:
What about it?

Reply:
it does not add a new element to the store right?

Reply:
ah yes, corrected